mod-source-record-manager-server/src/main/resources/templates/db_scripts/create_get_record_processing_log_function.sql

```diff
@@ -0,0 +1,99 @@
+-- Script to create function to get data import record processing log entries (recordProcessingLogDto).
+CREATE OR REPLACE FUNCTION get_record_processing_log(jobExecutionId uuid, recordId uuid)
+    RETURNS TABLE(job_execution_id uuid, source_id uuid, source_record_order integer, title text, source_record_action_status text, source_entity_id uuid, source_entity_hrid text, source_entity_error text, instance_action_status text, instance_entity_id uuid, instance_entity_hrid text, instance_entity_error text, holdings_action_status text, holdings_entity_id uuid, holdings_entity_hrid text, holdings_entity_error text, item_action_status text, item_entity_id uuid, item_entity_hrid text, item_entity_error text, order_action_status text, order_entity_id uuid, order_entity_hrid text, order_entity_error text, invoice_action_status text, invoice_entity_id uuid, invoice_entity_hrid text, invoice_entity_error text)
+AS $$
+BEGIN
+    RETURN QUERY EXECUTE format('
```

Comment:
```suggestion
    RETURN QUERY
```
it looks like we can do without dynamic command, hence here we can avoid redundant format() function call and directly write a query.